.github/workflows/macos.yml

```
@@ -449,7 +449,7 @@ jobs:
       - name: 'install autotools'
         if: ${{ matrix.build == 'autotools' }}
         run: |
-          echo autoconf automake libtool openssl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+          echo autoconf automake libtool openssl libpsl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
```

```suggestion
          echo autoconf automake libtool openssl | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
```

It works with autotools now, but cmake doesn't pick it up and it's intentional.
The 'combinations' jobs are meant to be minimal and match between autotools and cmake. So I think the correct solution is to use `--without-libpsl` here.